policy_packs/azure/network/check_virtual_networks_have_flowlogs_enabled/main.tf

```diff
@@ -0,0 +1,5 @@
+resource "turbot_policy_pack" "main" {
+  title       = "Check Azure Virtual Networks Have Flow Logs Enabled"
+  description = "Ensure that logging is enabled for Azure Virtual Networks to monitor traffic and maintain security compliance."
+  akas        = ["azure_network_check_virtual_networks_have_flowlog_enabled"]
```

Comment:
I think the aka would be `azure_network_enforce_virtual_networks_have_flowlog_enabled` if we include the `Enforce` option too?